_config.yml

```diff
@@ -92,7 +92,9 @@ defaults:
       type: "events"
     values:
       layout: "event"
+      namespace: event
```

Comment:
Just so I don't miss anything, these namespaces aren't used yet right? or are they for generating the breadcrumb? Because even a null namespace would confirm this is not an index page, the only time you use namespace logic in this PR? For other cases like translation, this would still be useful so can keep